### ISSUE TITLE
20bn website 503 Service Temporarily Unavailable

### ISSUE THREAD

**Issue by JiaMingLin** [TwentyBN/sense#184]
Hi, as the title, the 20bn official website is not available for me.
Is there any problem?

**Comment by nzx9** [comment]
Yeah, Because of this, datasets are inaccessible. Is there any other way to access datasets?

**Comment by sneheshs** [comment]
+1 for accessing the Jester dataset

**Comment by davideferrari95** [comment]
I also have this issue, how can I access to the pre-trained weights ?

**Comment by alberttjc** [comment]
Would like to raise this issue, should we request the training weights from previous projects? Or should we contact you directly to request these files? Let us know if we should contact anyone else to escalate this inqury.

Hope to hear from you soon!

**Comment by slowlyC** [comment]
> I also have this issue, how can I access to the pre-trained weights ?

Did you get the weights?

**Comment by davideferrari95** [comment]
> 
> 
> > I also have this issue, how can I access to the pre-trained weights ?
> 
> Did you get the weights?

Unfortunately not yet

**Comment by shadow150519** [comment]
Can anybody  share the pre-trained weights

**Comment by JonyJiang123** [comment]
Can anybody share the pre-trained weights +1

**Comment by davideferrari95** [comment]
I think sharing requires permission from the company as it is licensed material. I contacted some of the Twenty Billion Neurons managers and they gave me an email (datasets.20bn@qti.qualcomm.com) to write to request access to the dataset but to date no response.

**Comment by MoonBunnyZZZ** [comment]
> I think sharing requires permission from the company as it is licensed material. I contacted some of the Twenty Billion Neurons managers and they gave me an email ([datasets.20bn@qti.qualcomm.com](mailto:datasets.20bn@qti.qualcomm.com)) to write to request access to the dataset but to date no response.

do you get response?

**Comment by davideferrari95** [comment]
> > I think sharing requires permission from the company as it is licensed material. I contacted some of the Twenty Billion Neurons managers and they gave me an email ([datasets.20bn@qti.qualcomm.com](mailto:datasets.20bn@qti.qualcomm.com)) to write to request access to the dataset but to date no response.
> 
> do you get response?

I sent two emails with their CEO in CC but unfortunately I not get any response. I don't know how to access the database.
If someone owns those files and get the permission for sharing them it would be awesome but we need someone to reply to this issue.

**Comment by MoonBunnyZZZ** [comment]
> > > I think sharing requires permission from the company as it is licensed material. I contacted some of the Twenty Billion Neurons managers and they gave me an email ([datasets.20bn@qti.qualcomm.com](mailto:datasets.20bn@qti.qualcomm.com)) to write to request access to the dataset but to date no response.
> > 
> > 
> > do you get response?
> 
> I sent two emails with their CEO in CC but unfortunately I not get any response. I don't know how to access the database. If someone owns those files and get the permission for sharing them it would be awesome but we need someone to reply to this issue.

So do I

**Comment by sudonto** [comment]
Any updates on this inaccesible site issue?

**Comment by KeonAcute** [comment]
Have anyone get the pre-trained weights？

**Comment by koralm** [comment]
Have anyone get the pre-trained weights and can share via email ?

**Comment by davideferrari95** [comment]
> Have anyone get the pre-trained weights and can share via email ?

I tried to contact everyone but the site still unavailable and writing to the Qualcomm address given to me to request those files  is useless, they don't reply.

I think the only solution is that someone shares those files via mail...

**Comment by sudonto** [comment]
> 

Hi @davideferrari95, you may train your model by yourself. If you dont have the dataset, you may visit https://developer.qualcomm.com/software/ai-datasets to get the desired one.

**Comment by davideferrari95** [comment]
> 

Hi, I have already downloaded the Jester dataset. So, if I have understood, you suggest to follow the [Build Your Own Classifier with SenseStudio](https://github.com/quic/sense#build-your-own-classifier-with-sensestudio) guide and re-build the jester dataset, right ? 
I will try and give a feedback in the next days, thanks for the suggestion.

**Comment by aureosun** [comment]
@all ,Have anyone get the pre-trained weights and can share via email ?
